src/lib/FreeC/IR/Subterm.hs

```diff
@@ -101,91 +94,80 @@ class Pretty a => Subterm a where
 -- | Expressions have subterms.
 instance Subterm IR.Expr where
   -- | Gets the direct child expression nodes of the given expression.
-  childTerms (IR.App         _ e1   e2 _) = [e1, e2]
-  childTerms (IR.TypeAppExpr _ expr _  _) = [expr]
-  childTerms (IR.If _ e1 e2 e3 _        ) = [e1, e2, e3]
-  childTerms (IR.Case   _ expr alts _   ) = expr : map IR.altRhs alts
-  childTerms (IR.Lambda _ _    expr _   ) = [expr]
-  childTerms (IR.Con _ _ _              ) = []
-  childTerms (IR.Var _ _ _              ) = []
-  childTerms (IR.Undefined _ _          ) = []
-  childTerms (IR.ErrorExpr  _ _ _       ) = []
-  childTerms (IR.IntLiteral _ _ _       ) = []
-  childTerms (IR.Let _ binds e _        ) = e : map IR.bindExpr binds
+  childTerms (IR.App _ e1 e2 _) = [e1, e2]
+  childTerms (IR.TypeAppExpr _ expr _ _) = [expr]
+  childTerms (IR.If _ e1 e2 e3 _) = [e1, e2, e3]
+  childTerms (IR.Case _ expr alts _) = expr : map IR.altRhs alts
+  childTerms (IR.Lambda _ _ expr _) = [expr]
+  childTerms (IR.Con _ _ _) = []
+  childTerms (IR.Var _ _ _) = []
+  childTerms (IR.Undefined _ _) = []
+  childTerms (IR.ErrorExpr _ _ _) = []
+  childTerms (IR.IntLiteral _ _ _) = []
+  childTerms (IR.Let _ binds e _) = e : map IR.bindExpr binds
```

Comment:
This is because Floskell would have to insert too many spaces in order to align the matches (it is currently configured to insert a padding of at most 10 spaces). This has now been mentioned multiple times (see <https://github.com/FreeProving/language-coq/pull/5#discussion_r472385599> and <https://github.com/FreeProving/haskell-src-transformations/pull/44#discussion_r472861300>). However for the reason I stated in <https://github.com/FreeProving/haskell-src-transformations/pull/44#discussion_r472946438>, I'm not sure whether we should actually increase the alignment limit. Maybe we should even disable alignment of function rules to prevent confusion... :thinking:

Reply:
So, if I understand that correctly, the main problem is that Floskell doesn't try to group matches and align the right-hand sides differently for each group, like Brittany does, but either aligns all right-hand sides to the same position or doesn't align any right-hand sides.
I think this kind of alignment is very useful in places like this one, so we should increase this padding configuration. I would suggest a maximum of 20 to 30 spaces, unless that somehow causes other places to look weird.

Reply:
> So, if I understand that correctly, the main problem is that Floskell doesn't try to group matches and align the right-hand sides differently for each group, like Brittany does, but either aligns all right-hand sides to the same position or doesn't align any right-hand sides.

Exactly.

> I would suggest a maximum of 20 to 30 spaces, unless that somehow causes other places to look weird.

The example in <https://github.com/FreeProving/haskell-src-transformations/pull/44#discussion_r472946438> uses a limit of 20 and it kind of looks weird to me. Nonetheless, I've now increased the limit to 20 spaces (see 3eb9c0e90238806daddb048c0ce51891a0c0125e) such that we can see how that would impact the overall readability of the code.